### ISSUE TITLE
[BUG] `// ignore_for_file: type=lint` seems not to work for `check-unnecessary-nullable`

### ISSUE THREAD

**Issue by fzyzcjy** [dart-code-checker/dart-code-metrics#978]
* **Dart code metrics version:** latest
* **Dart sdk version:** latest

**Please show your full configuration:**

Hi thanks for the library!

I use the quite helpful feature `check-unnecessary-nullable` to tidy the code, but I want it to ignore some files. Those files have `// ignore_for_file: type=lint` inside them, but seems that DCM does not take this comment into consideration.

<details>
<summary>Configuration</summary>

```yaml

```

</details>

**What did you do? Please include the source code example causing the issue.**

```dart

```

**What did you expect to happen?**

**What actually happened?**

**Are you willing to submit a pull request to fix this bug?**

**Comment by incendial** [comment]
@fzyzcjy I'm not sure that our commands should take `// ignore_for_file: type=lint` coz it's not actually a lint. You can use // ignore_for_file: unnecessary-nullable` https://github.com/dart-code-checker/dart-code-metrics/blob/master/lib/src/analyzers/unnecessary_nullable_analyzer/unnecessary_nullable_analyzer.dart#L34. Will this be enough for you?

**Comment by fzyzcjy** [comment]
Well this bug comes from some .g.dart, and I raise it b/c another bug of ignoring certain suffix.

Anyway I have workaround both bugs by wrapping the DCM w/ a python script :)
